pandas/tests/frame/methods/test_reset_index.py

```diff
@@ -374,11 +366,6 @@ def test_reset_index_multiindex_columns(self):
         result = df[["B"]].rename_axis("A").reset_index()
         tm.assert_frame_equal(result, df)
 
-        # GH#16120: already existing column
-        msg = r"cannot insert \('A', ''\), already exists"
```

Comment:
does this work now?

Reply:
My mistake, yes it does.
I have converted it to a test and added a futher test in series/methods